addons/knowledge/static/src/components/permission_panel/permission_panel.js

```diff
@@ -0,0 +1,334 @@
+/** @odoo-module **/
+
+import session from 'web.session';
```

Comment:
all these imports starting with `web` are deprecated. Please try to use the new code (starting with `@web`)